build/lib/rabies/analysis_pkg/diagnosis_pkg/diagnosis_functions.py

```
@@ -0,0 +1,561 @@
+    ax0_f.set_xlabel('Frequency (Hz)', fontsize=20)
+    ax0_f.set_ylabel('Power (a.u.)', fontsize=20)
+
+    # disable function
+    regional_grayplot=False
+    if regional_grayplot:
+        
+        from mpl_toolkits.axes_grid1 import make_axes_locatable
+        divider = make_axes_locatable(ax0)
+        
+        im, slice_alt, region_mask_label = grayplot_regional(
+            bold_file, mask_file_dict, fig, ax0)
```

Ruff (0.14.1)

Undefined name `bold_file`

(F821)